ui/entry-perf.js

```diff
@@ -22,6 +23,7 @@ require('mousetrap');
 require('bootstrap/dist/js/bootstrap');
 require('angular-ui-bootstrap');
 require('numeral');
+require('metrics-graphics');
```

Comment:
I think the direct includes are redundant with the angular fixture?

(Sadly the current includes set a bad example - they need cleaning up after the initial switch to Webpack)